### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by hr98w**

### CODE REVIEW

**Comment by ultratwo** [comment]
typo farmer. do not merge please

File: rlp/rlpgen/gen.go [hunk 1]
```diff
@@ -158,7 +158,7 @@ type op interface {
 // basicOp handles basic types bool, uint*, string.
 type basicOp struct {
 	typ           types.Type
-	writeMethod   string     // calle write the value
+	writeMethod   string     // call write the value
```
Comment (karalabe) [inline rlp/rlpgen/gen.go:161]: @fjl What should be here?

**Comment by karalabe** [comment]
> typo farmer. do not merge please

Myeah, true, but the PR does fix the typos...

File: rlp/rlpgen/gen.go (hunk 1 above)
Comment (fjl) [reply rlp/rlpgen/gen.go:161]: maybe "EncoderBuffer writer method name"

**Review by karalabe — APPROVED** [review]
SGTM

**Pull request merged**